raiden/utils/typing.py

```diff
@@ -105,7 +105,7 @@
 T_PaymentAmount = int
 PaymentAmount = NewType("PaymentAmount", T_PaymentAmount)
 
-T_InvoiceAmount = int
+T_InvoiceAmount = float
```

Comment:
afaik this is _only_ used as a type hint, but it's still good to change it

(thanks @mauricioirace)

Reply:
(you're welcome!)